Snake/BusinessLayer/GameSnake.cs

```diff
@@ -8,14 +8,19 @@ namespace BusinessLayer
 {
     public class GameSnake
     {
-        Direction DirectionFacing;
+        Direction directionFacing;
         private int snakeLength;
         private List<Point> currentPosition;
 
+        public Direction getDirectionFacing()

```

Comment:
consider making it a property with a public get and a private set instead of a fuction